files/usr/bin/patch_db.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#mkdir -p /data/postgresql/14/main/data
+#mkdir -p /srv/postgresql/14/main/conf
+#mv /etc/postgresql/14/main/* /srv/postgresql/14/main/conf
+#rm -rf /srv/postgresql/14/main
+#ln -s /srv/postgresql/14/main/conf /etc/postgresql/14/main
+sed -i -e 's/host    all             all             127.0.0.1\/32            scram-sha-256/host    all             all             127.0.0.1\/32            trust/g'  /etc/postgresql/14/main/pg_hba.conf
```

Comment:
this will fix the `Error: SASL: SCRAM-SERVER-FIRST-MESSAGE: client password must be a string` error because Unifi connects without a password and even in "defaults.yaml" file `password = null`